core/core.go

```diff
@@ -44,6 +43,7 @@ const (
 	peerConnectTimeout         = 60 * time.Second
 	checkNewAddrInterval       = 20 * time.Second
 	expirationPollingInterval  = 50 * time.Millisecond
+	version                    = "development"
```

Comment:
Note: We'll need to update the publish script once it merges.

Reply:
We also have to keep in mind that right now the string is "development" but after the next release, it will change to e.g., "v2.0.0-beta". We might have to update our script to work for both cases.

Reply:
Changed the script and it uses regexes now :)